conf/broker.conf

```diff
@@ -102,6 +102,12 @@ tlsAllowInsecureConnection=false
 # Using a value of 0, is disabling unackeMessage limit check and consumer can receive messages without any restriction 
 maxUnackedMessagesPerConsumer=50000
 
+# Max number of unacknowledged messages allowed per shared subscription. Broker will stop dispatching messages to
+# all consumers once, this limit reaches until consumer starts acknowledging messages back and unack count reaches
+# to maxUnackedMessagesPerDispatcher/2. Using a value of 0, is disabling unackedMessage-limit check and dispatcher
+# can dispatch messages without any restriction
+maxUnackedMessagesPerDispatcher=200000
```

Comment:
> Then would it be feasible to have an efficient way (maybe with relaxed enforcement) to add a per-broker limit?

But then it raises question:
1. If we have per-broker limit then broker blocks dispatching to all topics once it reaches that limit: So, if only few of the topics are misbehaving then will broker punish other topics as well and blocks dispatching on other topics as well?
2. once, misbehaving-client acks limit/2 msgs -> broker should notify all dispatchers to dispatch and there could be possibility that again misbehave-client made broker to reach that limit and again dispatching will be blocked. 

So, per-broker limit may not be good idea??